terraform/destroy.sh

```diff
@@ -26,7 +26,7 @@ SCRIPT_DIR=$(realpath $(dirname "$0"))
 cd $SCRIPT_DIR
 
 # find the stackdriver sandbox project id
-found=$(gcloud projects list --filter="name:stackdriver-sandbox-*" --format="value(projectId)")
+found=$(gcloud projects list --filter="id:stackdriver-sandbox-* AND name='Stackdriver Sandbox Demo'" --format="value(projectId)")
```

Comment:
revert this

Reply:
Oh, why is that? Shouldn't we also filter the name for demo?

Reply:
@Daniel-Sanche

Reply:
sorry, I thought this was another accidental overwrite. This can be kept